src/plasma/plasma.c

```diff
@@ -17,3 +17,9 @@ bool plasma_object_ids_distinct(int num_object_ids, object_id object_ids[]) {
   }
   return true;
 }
+
+void warn_if_failed(int status, int client_sock) {
+  if (status < 0) {
+    LOG_WARN("Failed to send a message to client on fd %d", client_sock);
+  }
+}
```

Comment:
I feel like this helper function is a bit of an overkill. Esp. given that its utility is limited, e.g. it couldn't be used in other places in this same PR where you want to do more than just log a warning message on socket i/o failure. Why not just stick with ``if ( command() < 0) { LOG_WARN(); /* other stmts*/ }``